### PR TITLE
Datastore supabase

### PR DESCRIPTION
**Pull request by HanniOfHyrule**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,11 +8,12 @@ Housemeter is a tool to archive and evaluate all meter readings in the house.
 
 - you can filter the measurements types with the dropdown
 - you can add new measurements
+- you can filter the mearurementss
```
Comment (domnikl) [inline README.md:11]: `s` again 😆

File: README.md [hunk 2]
```diff
@@ -8,11 +8,12 @@ Housemeter is a tool to archive and evaluate all meter readings in the house.
...
 
 ## It look like this
 
-![ScreenShot](/screenshot/Housemeter.png)
+![ScreenShot](/Housemeter.png)
```
Comment (domnikl) [inline README.md:15]: Why is it changed again?

File: src/SupabaseClient.js [hunk 3]
```diff
@@ -4,3 +4,15 @@ export const supabase = createClient(
   "https://burmmhznjahkpzyzltgg.supabase.co",
   process.env.REACT_APP_SUPABASE_API_KEY
 );
+export async function addReading(measurement) {
+  const { data, error } = await supabase
+    .from("housemeter")
+    .insert([measurement]);
+  console.log(error, data);
```
Comment (domnikl) [inline src/SupabaseClient.js:11]: remove unnecessary `console.log`

File: README.md (hunk 1 above)
Comment (HanniOfHyrule) [reply README.md:11]: ✅

File: README.md (hunk 2 above)
Comment (HanniOfHyrule) [reply README.md:15]: i don´t know! kinky!

File: src/App.js [hunk 4]
```diff
@@ -5,7 +5,7 @@ import Login from "./Login";
 import { supabase } from "./SupabaseClient";
 
 function App() {
-  const [user, setUser] = useState(null);
+  const [user, setUser] = useState("bla");
```
[the remark this answers is not in the document]
Comment (HanniOfHyrule) [reply src/App.js:8]: 🙈

File: src/components/MeasurementsTable.js [hunk 5]
```diff
@@ -56,6 +42,7 @@ const MeasurementsTable = (props) => {
               <select
                 className={classes.meterTypeSelect}
                 onChange={(e) => setFilter(e.target.value)}
+                key={Number.toString()}
```
[the remark this answers is not in the document]
Comment (HanniOfHyrule) [reply src/components/MeasurementsTable.js:45]: right, it doesn´t need a key.

File: src/App.js [hunk 6]
```diff
@@ -5,7 +5,7 @@ import Login from "./Login";
 import { supabase } from "./SupabaseClient";
 
 function App() {
-  const [user, setUser] = useState(null);
+  const [user, setUser] = useState("null");
```
Comment (HanniOfHyrule) [inline src/App.js:8]: only `null`

File: src/SupabaseClient.js [hunk 7]
```diff
@@ -4,3 +4,23 @@ export const supabase = createClient(
...
+    console.log(data,error)
+}
+
+export async function getReadings() {
+  const { data, error } = await supabase.from("housemeter").select("*");
+  console.log(error)
+  return data;
+}
+
+// export async function deleteData() {
```
Comment (domnikl) [inline src/SupabaseClient.js:20]: shouldn't be here

File: src/components/MeasurementsTable.js [hunk 8]
```diff
@@ -73,8 +59,8 @@ const MeasurementsTable = (props) => {
             <MeasurementsTableItem
               item={reading}
               filter={filteredReadings}
-              key={reading.id}
               className={classes.tableContainer}
+              key={v4}
```
Comment (domnikl) [inline src/components/MeasurementsTable.js:63]: The key should be set to `reading.id`

**Review by domnikl — CHANGES_REQUESTED** [review]

**Review by domnikl — APPROVED** [review]

**Pull request merged**